demos/immersed_fem/immersed_fem.py.rst

```diff
@@ -0,0 +1,115 @@
+Creating Firedrake-compatible meshes in Gmsh
+============================================
+
+The purpose of this demo is to summarize the
+key structure of a ``gmsh.geo`` file that creates a
+Firedrake-compatible mesh. For more details about Gmsh, please
+refer to the `Gmsh documentation <http://gmsh.info/#Documentation>`_.
+
+The first thing we define are four corners of a rectangle.
+We specify the x,y, and z(=0) coordinates, as well as the target
+element size at these corners (which we set to 0.5). ::
+
+  Point(1) = {-6,  2, 0, 0.5};
+  Point(2) = {-6, -2, 0, 0.5};
+  Point(3) = { 6, -2, 0, 0.5};
+  Point(4) = { 6,  2, 0, 0.5};
+
+Then, we define 5 points to describe a circle. ::
+
+  Point(5) = {   0,    0, 0, 0.1};
+  Point(6) = { 0.5,    0, 0, 0.1};
+  Point(7) = {-0.5,    0, 0, 0.1};
+  Point(8) = {   0,  0.5, 0, 0.1};
+  Point(9) = {   0, -0.5, 0, 0.1};
+
+Then, we create 8 edges: 4 for the rectangle and 4 for the circle.
+Recall that the Gmsh command ``Circle`` requires the arc to be
```

Comment:
Recall is the wrong word here (it suggests to the reader they missed something above), "Note" might be better.

Reply:
changed, thanks